src/ui/pages/home.page.ts

```diff
@@ -0,0 +1,32 @@
+import { PortalPage } from './portal.page.js';
+
+class HomePage extends PortalPage {
+  //   uniqueElement = '//strong[.="AQA User"]';
+  uniqueElement = 'strong';
+
+  readonly 'Orders button' = "//button[@id='orders-from-home']";
+  readonly 'Products button' = "//button[@id='products-from-home']";
+  readonly 'Customers button' = "//button[@id='customers-from-home']";
+
+  async openOrders() {
+    await $(this['Orders button']).click();
+  }
```

Comment:
Используем методьі basePage для действий с єлементами
```suggestion
  async openOrders() {
    await this.click(this['View details module button']('orders'));
  }
```